cmd/system-probe/modules/oom_kill_probe.go

```diff
@@ -36,16 +36,18 @@ var OOMKillProbe = module.Factory{
 	},
 }
 
-var _ module.Module = &oomKillModule{}
+var _ module.Module = &oomKillModule{
+	lastCheck: atomic.NewInt64(0),
```

Comment:
This variable is unused, and is only for type checks. I think you want this initialization on line 35 instead.